casper/src/main/scala/coop/rchain/casper/genesis/contracts/ProofOfStake.scala

```diff
@@ -15,6 +15,7 @@ final case class ProofOfStake(
     quarantineLength: Int,
     numberOfActiveValidators: Int,
     posMultiSigPublicKeys: List[String],
+    posVaultPubKey: String,
```

Comment:
Keep `posMultiSigPublicKeys` and `posMultiSigQuorum` parameters together. They are part of the same functionality.